db/migrate/20160114121308_remove_link_fields.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveLinkFields < ActiveRecord::Migration
```

Comment:
I was thinking we could do that script in a separate PR and deploy that one first, rather than splitting this one up. Would that be simpler?

We haven't got around to writing the script itself yet, but @Davidslv knows about it.